include/syntax/swift

```diff
@@ -0,0 +1,412 @@
+re2c:yych:conversion = 0;
+re2c:yych:literals = hex;
+re2c:yych:emit = (.code_model.recursive_functions ? 0 : 1);
+re2c:yybm = "yybm";
+re2c:yybm:hex = 0;
+re2c:yyfill = "";
+re2c:yystable = ""; // deprecated
+re2c:header = "";
+re2c:eof = -1;
+re2c:sentinel = -1;
+re2c:yyfill:enable = 1;
+re2c:yyfill:parameter = 1;
+re2c:yyfill:check = 1;
+re2c:tags = 0;
+re2c:tags:prefix = "yyt";
+re2c:captures = 0;
+re2c:captvars = 0;
+re2c:posix-captures = 0;
+re2c:posix-captvars = 0;
+re2c:invert-captures = 0;
+re2c:cond:abort = 1;
+re2c:cond:prefix = "yyc_";
+re2c:cond:enumprefix = "yyc";
+re2c:cond:divider@cond = "@@";
+re2c:cond:goto@cond = "@@";
+re2c:state:abort = 1;
+re2c:state:nextlabel = 0;
+re2c:bit-vectors = 0;
+re2c:debug-output = 0;
+re2c:computed-gotos = 0;
+re2c:computed-gotos:threshold = 9;
+re2c:computed-gotos:relative = 0;
+re2c:nested-ifs = 0;
+re2c:case-insensitive = 0;
+re2c:case-inverted = 0;
+re2c:case-ranges = 1;
+re2c:unsafe = 0;
+re2c:monadic = 0;
+re2c:encoding:ebcdic = 0;
+re2c:encoding:utf32 = 0;
+re2c:encoding:ucs2 = 0;
+re2c:encoding:utf16 = 0;
+re2c:encoding:utf8 = 0;
+re2c:encoding-policy = ignore;
+re2c:empty-class = match-empty;
+re2c:indent:string = "  ";
+re2c:indent:top = 0;
+re2c:label:prefix = "yy";
+re2c:label:yyfill = "";
+re2c:label:yyloop = "yyl";
+re2c:label:yyNext = "";
+re2c:label:start = 0;
+
+
+// mutable code configuration --------------------------------------------------
+
+re2c:YYBACKUP = "yybackup";
+re2c:YYBACKUPCTX = "yybackupctx";
+// Conditions are assigned to `yystate` in loop/switch mode (which is UInt),
+// but stored state is Int, so try to default to most appropriate type.
+re2c:YYCONDTYPE = (.code_model.loop_switch & !.storable_state ? "UInt" : "Int");
+re2c:YYCOPYMTAG = sigil "{lhs} = " sigil "{rhs}";
+re2c:YYCOPYSTAG = sigil "{lhs} = " sigil "{rhs}";
+re2c:YYCTYPE = "UInt8";
+re2c:YYCURSOR = (.api.record ? yyrecord ".") "yycursor";
+re2c:YYDEBUG = "yydebug";
+re2c:YYFILL = "yyfill";
+re2c:YYGETACCEPT = sigil "{var}";
+re2c:YYGETCOND = "yygetcond";
+re2c:YYGETSTATE = "yygetstate";
+re2c:YYINPUT = (.api.record ? yyrecord ".") "yyinput";
+re2c:YYLESSTHAN = "yylessthan";
+re2c:YYLIMIT = (.api.record ? yyrecord ".") "yylimit";
+re2c:YYMARKER = (.api.record ? yyrecord ".") "yymarker";
+re2c:YYMAXFILL = "yymaxfill";
+re2c:YYMAXNMATCH = "yymaxnmatch";
+re2c:YYMTAGN = "yymtagn";
+re2c:YYMTAGP = "yymtagp";
+re2c:YYPEEK = "yypeek";
+re2c:YYRESTORE = "yyrestore";
+re2c:YYRESTORECTX = "yyrestorectx";
+re2c:YYRESTORETAG = "yyrestoretag";
+re2c:YYSETACCEPT = sigil "{var} = " sigil "{val}";
+re2c:YYSETCOND = "yysetcond";
+re2c:YYSETSTATE = "yysetstate";
+re2c:YYSHIFT = "yyshift";
+re2c:YYSHIFTSTAG = "yyshiftstag";
+re2c:YYSHIFTMTAG = "yyshiftmtag";
+re2c:YYSKIP = "yyskip";
+re2c:YYSTAGN = "yystagn";
+re2c:YYSTAGP = "yystagp";
+re2c:tags:expression = (.api.record ? yyrecord ".") sigil;
+re2c:tags:negative = (.api.generic ? "@@" : "-1");
+re2c:cond:divider = "/* *********************************** */";
```

Comment:
I think you can make it empty string, this is used in goto/label mode which is unsupported for Swift backend.

Reply:
Done.